release_images_inference.yml

```diff
@@ -63,3 +63,29 @@ release_images:
       example: False
       disable_sm_tag: False
       force_release: False
+  6:
+    framework: "pytorch"
+    version: "1.12.1"
+    arch_type: "x86"
+    customer_type: "ec2"
+    inference:
+      device_types: [ "cpu", "gpu" ]
+      python_versions: [ "py38" ]
```

Comment:
Is it `py310` or `py38`? Buildspec ref: https://github.com/aws/deep-learning-containers/blob/master/pytorch/inference/buildspec.yml#L37. Buildspec could look different for another release, but just want to ensure as a precautionary measure.

Reply:
That's the default buildspec for PT 2.0 that you are referring. PT 1.12 uses a different buildspec.